tools/bench_connect.c

```
@@ -0,0 +1,243 @@
+    free(tids);
+
+    int n = total_n;
+
+    /* Compute stats */
+    double sum_total = 0;
+    for (int i = 0; i < n; i++) sum_total += total_us[i];
+    double cps = n / wall_elapsed;
+
+    double *sorted_conn = malloc(n * sizeof(double));
+    double *sorted_total = malloc(n * sizeof(double));
+    memcpy(sorted_conn, connect_us, n * sizeof(double));
```

Cppcheck (2.20.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)